### PR TITLE
privacy statement updates from the original.

### PR DESCRIPTION
**Pull request by mtrojan-ub**

### CODE REVIEW

File: themes/tuefind/templates/content/privacy.phtml [hunk 1]
```diff
@@ -92,7 +101,9 @@
...
+    <p>Sie können jederzeit der Speicherung und Auswertung dieser Daten durch Matomo widersprechen (Link: <a href="https://vitruv.uni-tuebingen.de/piwik/index.php?module=CoreAdminHome&action=optOut&language=de&backgroundColor=&fontColor=&fontSize=&fontFamily=">https://vitruv.uni-tuebingen.de/piwik/index.php?module=CoreAdminHome&action=optOut&language=de&backgroundColor=&fontColor=&fontSize=&fontFamily=</a>). In diesem Fall wird in Ihrem Browser dauerhaft ein sog. Opt-Out-Cookie abgelegt, der Matomo veranlasst, keinerlei Daten für die Speicherung und Auswertung zu erheben. Sollten Sie dieses Cookie jedoch absichtlich oder unabsichtlich löschen, wird auch der Widerspruch gegen die Datenspeicherung und-auswertung aufgehoben und kann über den oben genannten Link erneuert werden.</p>
```
Comment (ruschein) [inline themes/tuefind/templates/content/privacy.phtml:105]: I think there should be a space after "und" and before the dash.

File: themes/tuefind/templates/content/privacy.phtml [hunk 2]
```diff
@@ -92,7 +101,9 @@
...
+    <p>Alternativ verfügen die meisten modernen Browser über eine so genannte „Do Not Track“-Option, mit der Sie Websites mitteilen, Ihre Nutzeraktivitäten nicht zu verfolgen. Matomo respektiert diese Option.</p>
```
Comment (ruschein) [inline themes/tuefind/templates/content/privacy.phtml:106]: The Duden recommends to use "sogenannte": https://www.duden.de/rechtschreibung/sogenannt

File: themes/tuefind/templates/content/privacy.phtml [hunk 3]
```diff
@@ -178,6 +189,6 @@
     <h2>Änderung der Datenschutzerklärung</h2>
     <p>Diese Datenschutzerklärung wird bei neuen gesetzlichen Vorgaben oder wesentlichen Änderungen des Funktionsumfangs unserer Webseiten mit Wirkung für die Zukunft aktualisiert. Wir empfehlen Ihnen daher, unsere Datenschutzerklärung in regelmäßigen zeitlichen Abständen durchzulesen.</p>
 
-    <!-- basierend auf Original-Datenschutzerklärung mit letzter Änderung 10.01.2019 -->
-    <p>Letzte Änderung: 05.06.2019</p>
+    <!-- basierend auf Original-Datenschutzerklärung mit letzter Änderung 16.03.2021 -->
```
Comment (ruschein) [inline themes/tuefind/templates/content/privacy.phtml:192]: Was that actually pushlished at that date?

**Review by ruschein — CHANGES_REQUESTED** [review]

File: themes/tuefind/templates/content/privacy.phtml (hunk 3 above)
Comment (mtrojan-ub) [reply themes/tuefind/templates/content/privacy.phtml:192]: Yes, the last change to the original has been made recently.

File: themes/tuefind/templates/content/privacy.phtml (hunk 1 above)
Comment (mtrojan-ub) [reply themes/tuefind/templates/content/privacy.phtml:105]: This paragraph is taken from the original.

File: themes/tuefind/templates/content/privacy.phtml (hunk 2 above)
Comment (mtrojan-ub) [reply themes/tuefind/templates/content/privacy.phtml:106]: This paragraph is taken from the original as well.

**Pull request merged**